tfhe/benches/integer/bench.rs

```diff
@@ -806,11 +808,21 @@ define_server_key_bench_scalar_default_fn!(
     display_name: add,
     rng_func: default_scalar
 );
+define_server_key_bench_scalar_default_fn!(
+    method_name: unsigned_overflowing_scalar_add_parallelized,
+    display_name: add,
```

Comment:
shouldn't the display_name be `overflowing_add` ?

Reply:
It would be more clear you're right.